modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/RoleResourceImpl.java

```diff
@@ -70,6 +71,17 @@ public void deleteOrganizationRoleUserAccountAssociation(
 			userAccountId, organization.getGroupId(), new long[] {roleId});
 	}
 
+	public void deleteRoleByExternalReferenceCodeUserAccountAssociation(
+			String roleExternalReferenceCode, Long userAccountId)
```

Comment:
the parameter name should be the same as in the base class file (I think are `String externalReferenceCode, Long userAccountId`